repl/src/main/scala/ammonite/repl/Main.scala

```diff
@@ -77,15 +80,17 @@ case class Main(predef: String = "",
     */
   def runScript(path: Path,
                 args: Seq[String],
-                kwargs: Map[String, String]): Res[Imports] = {
+                kwargs: Map[String, String],
+                storage: Storage): Res[Seq[ImportData]] = {
```

Comment:
You should be able to use the `storageBackend` from the case class here, and not need to pass it in yourself again